### PR TITLE
 YG-124-UX:Added style to the attendance module.

### PR DESCRIPTION
**Pull request by Shivani922**
# Related issues
- Added style to the attendance module

### CODE REVIEW

File: src/components/common/Header.jsx [hunk 1]
```diff
@@ -232,7 +233,7 @@ const Header = ({
 			<HeaderStyled
 				type="title"
 				style={headerObject.titleStyle ? headerObject.titleStyle : {}}>
-				{headerObject.title ? headerObject.title : getHeaderText()}
+				{headerObject.title ? '' : ''}
```
Comment (pepcus-mohit) [inline src/components/common/Header.jsx:236]: This change can impact all the headers on the other screen, Make sure other screen headers look good.

File: src/components/common/fields/InputField.jsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ import Tooltip from '../Tooltip/Tooltip';
 const TypographyStyled = styled(Typography)`
    color: ${getThemeProps('typography.titleFieldColor.color')}
    font-weight: bold !important;
-   display: block;
+	display: flex;
```
Comment (pepcus-mohit) [inline src/components/common/fields/InputField.jsx:17]: Make sure other input fields on registration page or correction page or login forms are not getting impacted by this.

**Review by pepcus-mohit — CHANGES_REQUESTED** [review]

File: src/components/common/Header.jsx (hunk 1 above)
Comment (Shivani922) [reply src/components/common/Header.jsx]: Resolved

File: src/components/common/fields/InputField.jsx (hunk 2 above)
Comment (Shivani922) [reply src/components/common/fields/InputField.jsx:17]: no it is not impacting anything. On member search, search filled was impacted but i fixed that also.

**Review by pepcus-mohit — APPROVED** [review]

**Pull request merged**